client/src/actions/index.ts

```diff
@@ -131,30 +126,6 @@ async function datasetMetadataFetchAndLoad(
   });
 }
 
-/**
```

Comment:
removing some MVP code. I'm sure there are more to delete, but that can be done in this ticket:

https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell-explorer/894

Reply:
assigned to myself - will take a look at this tomorrow and see what else can be removed